components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/rest/APICorsConfigurationDTO.java

```diff
@@ -25,6 +26,7 @@
 /**
  * CORS configuration for the APIDetailedDTO\n
  **/
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Add a comment to use of this property

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678